btk/measure.py

```diff
@@ -72,7 +74,7 @@ def basic_measure(batch, idx, channels_last=False, **kwargs):
     Returns:
             dict containing catalog with entries corresponding to measured peaks.
     """
-    if isinstance(batch["blend_images"], dict):
+    if "blend_images" not in batch.keys():
```

Comment:
Doesn't this always evaluate to false?

Reply:
What was wrong with the above approach? (was it too cryptic?)

Reply:
You're right, this is my fault ; I changed it when I had reversed the order for the surveys, but I ended up reversing it again so I should put it back as it was before.